app/src/main/java/com/android/voyageur/model/notifications/TripInviteRepositoryFirebase.kt

```diff
@@ -0,0 +1,196 @@
+package com.android.voyageur.model.notifications
+
+import com.google.firebase.firestore.AggregateSource
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.ListenerRegistration
+import com.google.firebase.firestore.SetOptions
+
+/**
+ * Used for displaying trip invites and notifications Appears as a parameter to the ${UserViewModel}
+ *
+ * @param db the firestore instance to use
+ * @param collectionPath the path to the collection of trip invites
+ * @constructor creates a new TripInviteRepositoryFirebase
+ * @property db the firestore instance to use
+ * @property collectionPath the path to the collection of trip invites
+ */
+class TripInviteRepositoryFirebase(private val db: FirebaseFirestore) : TripInviteRepository {
+  private val collectionPath = "friendRequests"
```

Comment:
The collection path should be updated to something more appropriate, such as "tripInvites", to better reflect the purpose of the data it stores. While avoiding hardcoded strings is generally a good practice for maintainability, it's less critical in this backend context. That said, extracting the collection path into a constant can still improve clarity and make future updates easier.

Reply:
You are right, I forgot to update the collection name, I will rename it asap.